content/rs/references/rladmin.md

```diff
@@ -0,0 +1,42 @@
+---
+Title: The rladmin Command-Line Interface (CLI)
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+---
+Redis Enterprise Software (RS) includes rladmin, a command-line interface (CLI)
+that you can use to do advanced administrative tasks.
+Some of these tasks you can also do through the management web UI.
+
+For example, with the rladmin CLI you can:
+
+- View the detailed status of the cluster including information about each node, each database, and each shard
+- Failover between a master and slave shard, or a master and slave endpoint
+- Migrate a specific shard or endpoint between nodes
+- Tune specific internal parameters of a database
+- Upgrade the version of a database to the most current version supported by the cluster
+
+## Using the rladmin CLI
+
+To open the rladmin CLI:
+
+1. Login to the RS host with an account that is either:
+    - a member of the **redislabs** OS group
+    - change user to root with command: `sudo su --`
```

Comment:
@Paz-Yanover Remove this line?

Reply:
@bmansheim - can you please the history of this change? I recall we had some requests to had the sudo. If there is no relevant history (and explanations), let's remove it

Reply:
@Paz-Yanover I don't see any request for this so I will remove it.